Makefile

```diff
@@ -275,6 +275,9 @@ $(KIND): ## Build kind into tools folder
 .PHONY: $(KUBECTL_BIN)
 $(KUBECTL_BIN): $(KUBECTL) ## Building kubectl from tools folder
 
+.PHONY: $(KUSTOMIZE_BIN)
+	$(KUSTOMIZE_BIN): $(KUSTOMIZE) ## Building Kustomize from tools folder
```

Comment:
maybe need to make it similar to the others, and remove the indentation?

Reply:
Ok I see... Let me try fixing it